src/main/java/core/basesyntax/controllers/CreateManufacturerController.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.controllers;
+
+import core.basesyntax.lib.Injector;
+import core.basesyntax.model.Manufacturer;
+import core.basesyntax.service.ManufacturerService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class CreateManufacturerController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("core.basesyntax");
+    ManufacturerService manufacturerService = (ManufacturerService)
+            injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/manufacturer/addManufacturer.jsp")
+                .forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String name = req.getParameter("name");
+        String country = req.getParameter("country");
+        System.out.println(name + " " + country);
+        if (country.replaceAll("[^a-zA-z]", "").length() != country.length()) {
+            req.setAttribute("message", "You entered a non-existent country");
+            req.getRequestDispatcher("/WEB-INF/views/manufacturer/addManufacturer.jsp")
+                    .forward(req, resp);
```

Comment:
if you still want to do some validation, let's add `return` after this line, so that our code would not go below catch block

or it is easier to just remove the validation